### DISCUSSION TITLE
My First NSPanel

### DISCUSSION THREAD

**Discussion by SDerbyshire** [Blackymas/NSPanel_HA_Blueprint#1191] [Show and tell]
Yesterday I sucessfully flashed my first NSPanel and got the excellent Blueprint configured and working !

Am loving this project , thank you Blackymas and all concerned for putting in the hours to make it possible.

I did the electronics part in my 'hobby room' and then installed the panel in the hall, it's working well.
![NS1](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/146989076/1c30cb58-c9e9-44b3-aff2-f1e9387c42dc)
![ns3](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/146989076/ddd9b5ac-4444-4633-8fa2-1d15cf8aac5d)
![ns2](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/146989076/735109c1-5311-42ad-8a3c-10d4fd627628)

I did have some trouble uploading the tft but the 'exit reparse' fixed this :-)

my only issue so far is that when i make changes in the automation to remove something like a Home Page Sensor it does not disappear from the screen? have tried several times with no luck.

I have two more NSPanels on the way to me so will be configuring those in the week.

thanks again !

Steve

**Comment by edwardtfn** [comment]
> my only issue so far is that when i make changes in the automation to remove something like a Home Page Sensor it does not disappear from the screen? have tried several times with no luck.

Sorry, I've just saw that.
If you still having this issue, could you please report as a bug in the [Issues](https://github.com/Blackymas/NSPanel_HA_Blueprint/issues) area?
